server/src/test/java/org/apache/druid/metadata/IndexerSqlMetadataStorageCoordinatorTestBase.java

```diff
@@ -322,6 +323,7 @@ protected DataSegment createSegment(Interval interval, String version, ShardSpec
                       .version(version)
                       .shardSpec(shardSpec)
                       .size(100)
+                      .loadSpec(ImmutableMap.of("hash", Objects.hash(interval, version, shardSpec)))
```

Comment:
Why is this needed? What does `hash` represent in `loadSpec`?

Reply:
Seemed easier this way because the segment id would be created after the segment has been created.

Reply:
Okay, that works too. Just add a one-line comment so that future readers know the intent of the hashing logic here.